app/views/container_topology/show.html.haml

```diff
@@ -24,56 +25,56 @@
   .legend
     %label#selected
     %div{'ng-if' => "kinds"}
-      %kubernetes-topology-icon{:kind => "ContainerReplicator", "tooltip-placement" => "bottom", "tooltip" => "{{legendTooltip}}"}
+      %kubernetes-topology-icon{tooltipOptions, :kind => "ContainerReplicator"}
```

Comment:
@himdel I changed to usage to include 2 stars (only in the usage, not the hash definition), and now the page fails to display with "wrong argument type String (expected Symbol)" - does it work for you?